authority/tls.go

```diff
@@ -546,7 +546,7 @@ func (a *Authority) GetTLSCertificate() (*tls.Certificate, error) {
 	}
 
 	// Create initial certificate request.
-	cr, err := x509util.CreateCertificateRequest("Step Online CA", sans, signer)
+	cr, err := x509util.CreateCertificateRequest(a.config.CommonName, a.config.DNSNames, signer)
```

Comment:
```suggestion
	cr, err := x509util.CreateCertificateRequest(a.config.CommonName, sans, signer)
```

This should use the (prepared) `sans` instead, so that IPv6 hostname representations are converted to their IP representation to be consistent with IPv4. It might've gotten mixed up in a merge.